_data/docs.yml

```
@@ -167,6 +167,18 @@ apis:
       legacy: 1
       stable: 1
       nightly: 1
+  rapidspdf:
+    name: RAPIDSMPF
+    path: rapidsmpf
+    desc: 'RAPIDS Multi-Process Foundation (rapidsmp) a collection of multi-gpu, distributed memory algorithms written in C++ and exposed to Python'
+    ghlink: https://github.com/rapidsai/rapidsmpf
+    cllink: https://github.com/rapidsai/rapidsmpf/blob/main/CHANGELOG.md
+    versions:
+      # enable or disable links; 0 = disabled, 1 = enabled
+      legacy: 1
+      stable: 1
+      nightly: 1
```

Legacy would be 25.02, stable would be 25.04. We are only publishing nightly (25.06) docs to my knowledge.
```suggestion
      legacy: 0
      stable: 0
      nightly: 1
```

---

Ah!  We don't have legacy and stable yet.  Should I follow up with another PR after 25.06 to change these back to 1 ?

---

We can add "stable" after the 25.06 release, and "legacy" after the 25.08 release.